src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -12,12 +12,13 @@ import { focusElement } from '../utilities/ui';
 import ReviewChapters from './ReviewChapters';
 import SubmitController from './SubmitController';
 
+// *** THIS COMPONENT ISN'T BEING USED ANYWHERE? ***
```

Comment:
I didn't want to remove this component with the "code caution" in place this week